src/Pages/3DPrintingConsole/3DConsole.js

```diff
@@ -194,7 +94,16 @@ export default class PrintConsole3D extends React.Component {
             }}
           />
           <br />
-          {this.search().map((item, key) => this.requestForm(item, key))}
+          {this.search().map((item, key) => (
```

Comment:
why is this a function call?

Reply:
Idk. I didn't write it.. but I'm assuming requestForm is returning all the 3D printing requests, filtered by the search function, and then mapped out.

Reply:
Yeah you are right.
```requestForm``` is the UI for each request-block that you see. And Calvin didn't write it.
------
My man Calvin, could you help me refactor this too.

Since, ```this.search()``` returns an array based on the search input. 
Instead of returning an array can we create a new variable

Make a new variable in state: ```requests: []```
The ```search()``` can 
```js 
this.setState( search )
 ``` 
instead of return (you will need to figure out where to call the search() function after each search)